src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/VideoLinkBookingsService.kt

```diff
@@ -2,31 +2,32 @@ package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service
 
 import jakarta.persistence.EntityNotFoundException
 import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.BookingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.VideoBookingSearchRequest
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.response.VideoLinkBooking
-import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.PrisonAppointmentRepository
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.ReferenceCodeRepository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.VideoAppointmentRepository
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.VideoBookingRepository
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.mapping.toModel
 
 @Service
+@Transactional(readOnly = true)
 class VideoLinkBookingsService(
   private val videoBookingRepository: VideoBookingRepository,
-  private val prisonAppointmentRepository: PrisonAppointmentRepository,
   private val referenceCodeRepository: ReferenceCodeRepository,
+  private val videoAppointmentRepository: VideoAppointmentRepository,
 ) {
-
   fun getVideoLinkBookingById(videoBookingId: Long): VideoLinkBooking {
     val booking = videoBookingRepository.findById(videoBookingId)
       .orElseThrow { EntityNotFoundException("Video booking with ID $videoBookingId not found") }
 
-    val prisonAppointments = prisonAppointmentRepository.findByVideoBooking(booking)
```

Comment:
We don't need to do this anymore.  The appointments can be pulled directly off the booking.